src/support.cc

```diff
@@ -383,7 +384,7 @@ void print_property(const property& prop) {
     } else if (prop.comparator() == ">=") {
       printf(" >= ");
     }
-    printf("%lu", prop.int_value());
+    printf("%" PRIu64, prop.int_value());
```

Comment:
Minor fix to use the right print format specifier, which is encoded by `PRIu64`. This is defined on each platform appropriate to what the compiler system requires.